nototools/noto_lint.py

```diff
@@ -1051,6 +1051,38 @@ def check_ul_unicode_range():
 
             check_ul_unicode_range()
 
+            if os2_table.panose.bFamilyType != 2:
+              warn("head/os2/panose/family", "OS/2",
+                   "Panose family value is %s but expected 2" %
+                   os2_table.panose.bSerifStyle)
+
+            expect_serif = noto_font.style == 'Serif' or noto_font.family in [
+                'Cousine', 'Tinos']
+            expected_serif_range_str = '2-10' if expect_serif else '11-15'
+            serif_val = os2_table.panose.bSerifStyle
+            is_serif = 1 < serif_val < 11
+            if serif_val == 1:
+              warn("head/os2/panose/serif", "OS/2",
+                   "Panose serif value is no_fit (1) but expected 0, %s" %
+                   expected_serif_range_str)
+            elif serif_val != 0 and expect_serif != is_serif:
+              warn("head/os2/panose/serif", "OS/2",
+                   "Panose serif value is %s but expected %s" %
+                   (serif_val, expected_serif_range_str))
+
+            expect_mono = noto_font.is_mono or noto_font.family == 'Cousine'
+            expect_mono_range_str = '9' if expect_mono else '2-4'
```

Comment:
With variation fonts that can have interpolated widths this gets a bit murkier.  Right now I'm keying off the font name to determine the expectations.  I guess as long as the values correspond to the name it's ok.

I'd prefer to submit this as is, then submit the updates to lint so that it can parse the font names, then look at what kinds of issues arise with condensed/semicondensed fonts and fix this then.